### DISCUSSION TITLE
Send from multiple email-adresses

### DISCUSSION THREAD

**Discussion by elmoumenyassine** [Budibase/budibase#7793] [Help]
Hi all,

I installed BUDIBASE via 1-Click-DIGITALOCEAN with a MYSQL database.

I have different departments and each has it's own email. How to set automations with diffrent sender email so the end customer will respond to the email directly.

Thanks All !

**Comment by melohagan** [comment]
Hey @elmoumenyassine 

I don't believe this can currently be done. There is an issue open here: https://github.com/Budibase/budibase/issues/4385

There's also a PR open here: https://github.com/Budibase/budibase/pull/7816
I might take a look at it to see if it's good to go!
